app/models/PregnancyCenter.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const mongoose = require('mongoose');
+
+const pregnancyCenterSchema = mongoose.Schema({
+    name: String,
+    address: String,
+    phone: String,
+    website: String
+})
+
+// create model using the schema
+const PregnancyCenter = mongoose.model('PregnancyCenters', pregnancyCenterSchema)
```

Comment:
Let's follow the naming convention `PregnancyCenterModel` so that it's clear that it is actually a model. That's my bad, I should have named it that initially. Also, let's name the collections plural and models singular. So `PregnancyCenters` is the collection and `PregnancyCenter` is the model. Does that make sense? Basically, this line would now be:
`const PregnancyCenterModel = mongoose.model('PregnancyCenter', pregnancyCenterSchema)`